code/game/objects/items/weapons/storage/belt.dm

```diff
@@ -6,18 +6,22 @@
 	item_state = "utility"
 	slot_flags = SLOT_BELT
 	attack_verb = list("whipped", "lashed", "disciplined")
+	var/use_item_overlays = 0 // Do we have overlays for items held inside the belt?
 
 /obj/item/weapon/storage/belt/update_icon()
-	cut_overlays()
-	for(var/obj/item/I in contents)
-		add_overlay("[I.name]")
+	if(use_item_overlays)
+		cut_overlays()
+		for(var/obj/item/I in contents)
+			add_overlay("[I.name]")

```

Comment:
children of welding tool won't show anything because their name is not "welding tool". If I take a hand labeller and label wirecutters they suddenly don't show an overlay when on the belt. If the wizard uses the wizard event that changes every object's name to have "of the bear +5" the items won't show either.